src/utils/rest-api.js

```diff
@@ -723,3 +723,41 @@ export function getLoadFlowParameters(studyUuid) {
             : response.text().then((text) => Promise.reject(text))
     );
 }
+
+function changeLineStatus(studyUuid, lineId, status) {
+    const changeLineStatusUrl =
+        getStudyUrl(studyUuid) +
+        '/network-modification/lines/' +
+        encodeURIComponent(lineId) +
+        '/status';
+    console.debug('%s with body: %s', changeLineStatusUrl, status);
+    return backendFetch(changeLineStatusUrl, { method: 'put', body: status });
```

Comment:
See comment about displayErrorMessageWithSnackbar to handle all errors

Reply:
Added displayInfoMessageWithSnackbar function to display messages